test/keras-to-hls.sh

```diff
@@ -41,10 +47,14 @@ function print_usage {
    echo "      Prints this help message."
 }
 
-while getopts ":x:c:sr:g:t:d:y:h" opt; do
+while getopts ":x:b:B:c:s:r:g:t:d:y:h" opt; do
```

Comment:
Instead of `:x:b:B:c:s:r:g:t:d:y:h` this needs to be `:x:b:B:c:sr:g:t:d:y:h` (no colon after `s`). See the output of the Jenkins log Keras to HLS stage, or when you execute these locally:
```
./keras-to-hls.sh  -s KERAS_3layer
No models specified.
./keras-to-hls.sh  -s KERAS_conv1d_small
No models specified.
./keras-to-hls.sh  -s KERAS_conv2d_model
No models specified.
./keras-to-hls.sh  -s jetTagger_Conv2D_Small
No models specified.
./keras-to-hls.sh  -s jetTagger_Conv2D_Small_NoBatchNorm
No models specified.
```

The `-s` option for io_stream takes no arguments, but adding the `:` after `s` expects an argument, so the model name is being treated as the argument to `-s`, hence the "No models specified" message.